ocaml/xe-cli/newcli.ml

```diff
@@ -233,16 +227,7 @@ let parse_args =
 
 let open_tcp_ssl server =
   let port = get_xapiport true in
-  debug "Connecting via%s stunnel to [%s] port [%d]%s\n%!"
-    (if !allow_ssl_legacy then " legacy-mode" else "") server port
-    (match !ciphersuites with None -> "" | Some c -> " with ciphersuites "^c);
-  Stunnel.set_legacy_protocol_and_ciphersuites_allowed !allow_ssl_legacy;
-  (match !ciphersuites with
-   | None -> ()
-   | Some c -> (* Use only the specified ones, none of Stunnel's built-in defaults. *)
-     Stunnel.set_good_ciphersuites c;
```

Comment:
I assume that you'll modify stunnel in xapi-stdext to have a good set of default good ciphersuites

Reply:
i will check that the 'good ciphersuites' in stdext are the same as in xapi

Reply:
Might be better to find a 'home' for the ciphersuite definition, and have every other daemon/etc. use that. For example this could be put somewhere in `xapi-idl`.

Reply:
this provides another reason for moving the bash scripts to ocaml